kiwi_lint/field_label.py

```diff
@@ -0,0 +1,27 @@
+from pylint import checkers
+from pylint import interfaces
+
+
+class FormFieldLabelChecker(checkers.BaseChecker):
+    """
+    Warns against defining labels in Form Field class constructors.
+    Labels must be placed in the HTML.
+    """
+    __implements__ = (interfaces.IAstroidChecker, )
+
+    name = 'form-field-label-checker'
+
+    msgs = {
+        'R4811': ('Do not use label in form Field class constructor.',
+                  'form-field-label-used',
+                  'Do not use label in form Field class constructor,'
+                  'place template label in HTML. See:'
+                  'https://github.com/kiwitcms/Kiwi/issues/738')
+    }
+
+    def visit_call(self, node):
+        if node.func.as_string().endswith('Field') and node.keywords:
+            for keyword in node.keywords:
```

Comment:
node.keywords is a list so you don't have to check it explicitly inside the if condition above. If it is empty the loop will not execute.

Reply:
node.keywords is set to `None` if there are no keyword args, unlike args positional arguments which is set to an empty list, so I think we need to explicit there. 

`Call(func=<Name.UserField l.15 at 0x1128bfa90>,
     args=[],
     keywords=None)`

Reply:
Grr, that's stupid on behalf of astroid. Leave it like this.